src/openrct2/paint/tile_element/Paint.Path.cpp

```diff
@@ -824,13 +824,13 @@ void path_paint(paint_session* session, uint16_t height, const TileElement* tile
 
         if (!track_design_save_contains_tile_element(tile_element))
         {
-            imageFlags = SPRITE_ID_PALETTE_COLOUR_1(PALETTE_46);
+            imageFlags = SPRITE_ID_PALETTE_COLOUR_1(EnumToFlag(FilterPaletteID::Palette46));
         }
     }
 
     if (session->ViewFlags & VIEWPORT_FLAG_HIGHLIGHT_PATH_ISSUES)
     {
-        imageFlags = SPRITE_ID_PALETTE_COLOUR_1(PALETTE_46);
+        imageFlags = SPRITE_ID_PALETTE_COLOUR_1(EnumToFlag(FilterPaletteID::Palette46));
```

Comment:
not a flag

Reply:
BTW, here the `imageFlags` name is a bit misleading.